classes/02class/exercises/c02-network03/README.md

```diff
@@ -0,0 +1,146 @@
+# AWS VPC + subnets
+
+Create a txt file under your folder name to add deliverables.
+
+Everything is to be done preferable on Sydney region `ap-southeast-2`
+
+### 1) VPC
+
+Create a new AWS VPC with:
+
+- CIDR block `/16`
+- Name: devopsacademy-vpc
+- No IPv6
+- Tenancy default
+
+#### Post details from the vpc created.
+
+You will notice that as soon as you create a VPC, a route table, default local routes and network acls are automatically deployed. More to that in the next exercises.
+
+
+### 2) Subnets
+
+Create 2 subnets (private and public) per Availability Zone (6 subnets in total) with following details:
+
+- name: <private/public>-<az-identifier> -- e.g.: `private-a` `private-b` 
+- VPC: use the VPC created on last exercise
+- AZ: refer to the name -- e.g.: subnet `private-a` will be `Availability Zone A` `ap-southeast-2a`
+- IPv4 CIDR: make it room for at least 250 hosts
+
+#### Post details from subnets created.
+
+
```

Comment:
Some posts are formatted with `####` and other are not. Can we have a single way?

Reply:
deliverables are ####, questions ###

Reply:
Ok.. Now you fixed. you had some without the ####
`- Post how did you accomplish that on AWS console.`.